scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
```

Comment:
:stop_sign: **Security Vulnerability**: Hardcoded API key detected. This is a critical security risk as it exposes sensitive information directly in the source code[^1].
```suggestion
// Remove hardcoded API key
// Use environment variables or a secure secret management system
const apiKey = process.env.API_KEY
if (!apiKey) {
    throw new Error('API key not found in environment variables')
}
```